bluemira/equilibria/grad_shafranov.py

```diff
@@ -79,8 +79,8 @@ def __call__(self, nx, nz):
         """
         d_x = (self.x_max - self.x_min) / (nx - 1)
         d_z = (self.z_max - self.z_min) / (nz - 1)
-        x = linspace(self.x_min, self.x_max, nx)
-        d_x2, d_z2 = d_x ** 2, d_z ** 2
+        x = linspace(self.x_min, self.x_max, nx) + 0.5 * d_x  # centroids
```

Comment:
This feels stranger and stranger the more I think about it... I think it might actually just be wrong to do this.